### ISSUE TITLE
verify changes doesn't do anything [bug]

### ISSUE THREAD

**Issue by ve6rah** [OpenZWave/Zwave2Mqtt#546]
**Version** 3.2.2

Build/Run method
- [ ] Docker
- [ ] PKG
- [ ] Manually built (git clone - npm install - npm run build ) 

Zwave2Mqtt version: 3.2.2
Openzwave Version: 1.6.974

**Describe the bug**
I have a dimmer module which takes time to transition from one state to another, but the value updates before the transition finishes. In HomeAssistant's zwave configuration this was easily solved by setting "refresh_value: true" with "delay: 3". In zwave2mqtt I've been told that setting "verify changes" in the device table of the gateway is the equivalent, however that toggle does nothing at all. 

**Expected behavior**
verify changes should verify that the changes have been properly applied to the node by refreshing the node after a short delay.

**Comment by robertsLando** [comment]
Could you show me how you configured that? You have to add the value to the gateway values table and enable the toggle

**Comment by ve6rah** [comment]
This is the configuration in the value table:
![Screenshot 2020-06-03 at 15 47 11](https://user-images.githubusercontent.com/16531454/83692652-a7216700-a5b1-11ea-8afe-816b515f9a80.png)
![Screenshot 2020-06-03 at 15 49 08](https://user-images.githubusercontent.com/16531454/83692736-cae4ad00-a5b1-11ea-9204-bce7155e99c2.png)

When the light is off and I send a level of 255 through MQTT the light starts it's fade in, and I get an immediate response of the level it has faded to so far (usually about 5 or so, but the number varies), This value is repeated a second time in MQTT as well. If the light is fully on, and I send a level of 0 through MQTT I only get a single response (instead of the 2 when turning on) but again it is almost instantaneous, and shows the value at that time, not the end value of the fade.

**Comment by ve6rah** [comment]
What I would really like is a setting like HA has where you basically tell it "after sending any command to this device, wait ___ seconds, and then request an updated value" This would make dealing with devices with slow or fading actions much more usable.

**Comment by chilicheech** [comment]
this is a feature i'm also interested in. i'm going to test the Verify Changes and see what it does.

**Comment by robertsLando** [comment]
> What I would really like is a setting like HA has where you basically tell it "after sending any command to this device, wait ___ seconds, and then request an updated value" This would make dealing with devices with slow or fading actions much more usable.

We could add a `Request updates` flag to do this. What do you think? @chilicheech @ve6rah

**Comment by ve6rah** [comment]
@robertsLando that sounds good. We should also be able to specify a delay for said request (ideal is the minimum time possible, but that varies per device, so making it user configurable would help).

On a side note, what is verify changes supposed to do if not this? I've seen no effect to this toggle.

**Comment by robertsLando** [comment]
AFAIK that flag should verify if a value change is a real value change, for example if the switch is on and I receive another value change with on value That is not emitted

**Comment by chilicheech** [comment]
Folks, since I needed this feature I tested the `Verify Changes` seeing and it does exactly that. From what I understand, when you set the dinner value, this seeing makes it so zwave keeps querying the value until it stops changing. So this effectively solves what were trying to do. @ve6rah can you try it out? It works well for me.

**Comment by ve6rah** [comment]
@chilicheech the whole reason I started this bug is because it specifically did not do that for me.

**Comment by chilicheech** [comment]
@ve6rah I see you're running Zwave2Mqtt version: 3.2.2, Openzwave Version: 1.6.974.. can you upgrade to the latest version and test again? Zwave2Mqtt version: 4.0.3, Openzwave Version: 1.6.1240

Another thing to note is I also have polling enabled for my dimmers. So, it may be that `Verify Changes` only works when polling is enabled.

**Comment by chilicheech** [comment]
Found some documentation on this: http://www.openzwave.com/dev/classOpenZWave_1_1Manager.html#ac26cc3c1b37cf068ac9eee23ef343c30

Per the docs it looks like it only polls 1 time after the value changes. If that's the case the we may want to implement something that will poll after a certain delay like home assistant does.

**Comment by ve6rah** [comment]
@chilicheech That is consistent with my experience. It polls once very shortly after the change, but as the dimmer is still in transition at that time, it gets that transitional value, and not the final value.

I believe the correct solution is as @robertsLando suggests.

As for updating, according to HASSOS, I am on the most recent version, so I'm not entirely certain how to upgrade further.

**Comment by chilicheech** [comment]
@ve6rah regarding upgrading, I meant upgrade ZWave2MQTT, not Hassio. Did you install Z2M as an Add On in Hassio? If so, then you're stuck at the version of the add on in there. It looks like they upgraded to 4.0.2 recently: https://github.com/hassio-addons/addon-zwave2mqtt

**Comment by ve6rah** [comment]
@chilicheech Yes, it's an addon to HassOS, the addon states it is at it's most recent version already. It appears that the add-on has recently updated to 4.0.2, however they haven't published a new release yet with that new version. I don't see any way on HassOS to update to an unreleased version of the addon, so I think I'm stuck for now.

**Comment by ve6rah** [comment]
I don't think anything had changed on this issue. Still seeing problems.

**Comment by robertsLando** [comment]
@ve6rah Could you give a try to: https://github.com/zwave-js/zwavejs2mqtt#why-zwavejs
